apps/src/code-studio/pd/application/principalApproval/PrincipalApprovalComponent.jsx

```diff
@@ -423,6 +425,12 @@ PrincipalApprovalComponent.getErrorMessages = data => {
     }
   });
 
+  if (Object.keys(formatErrors).length > 0) {
```

Comment:
It looks like we want to send this event with or without errors –– am I reading the spec correctly?

Reply:
Ah, I see your comment in the PR –– that makes sense, thanks for clarifying!